### PR TITLE
unit test of ParamCheckUtils.dubboBodyIsEmpty

### PR DESCRIPTION
**Pull request by marina432**
Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.dromara.soul.common.utils;
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java:1]: 加上apache的lincense

File: soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+
+public class ParamCheckUtilsTest {
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java:6]: 类加上注释 ，加上final

File: soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    @Test
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java:7]: 空一行

File: soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    public void testDubboBodyIsEmpty() {
+
```
Comment (yu199195) [inline soul-common/src/test/java/org/dromara/soul/common/utils/ParamCheckUtilsTest.java:9]: 不要空多余的行

**Comment by marina432** [comment]
ok

**Comment by dengliming** [comment]
@marina432 `git config user.email "xxxx"` rebase下设置下账号

**Review by yu199195 — APPROVED** [review]

**Pull request merged**